cli/commands/output.go

```diff
@@ -0,0 +1,87 @@
+package commands
+
+import (
+	"encoding/json"
+	"fmt"
+	"log"
+	"os"
+
+	"github.com/BattlesnakeOfficial/rules/client"
+)
+
+type GameExporter struct {
+	game          client.Game
+	snakeRequests []client.SnakeRequest
+	winner        SnakeState
+	isDraw        bool
+}
+
+type result struct {
+	WinnerID   string `json:"winnerId"`
+	WinnerName string `json:"winnerName"`
+	IsDraw     bool   `json:"isDraw"`
+}
+
+func (ge *GameExporter) FlushToFile(filepath string, format string) error {
+	var formattedOutput []string
+	var formattingErr error
+
+	// TODO: Support more formats
+	if format == "JSONL" {
+		formattedOutput, formattingErr = ge.ConvertToJSON()
+	} else {
+		log.Fatalf("Invalid output format passed: %s", format)
+	}
+
+	if formattingErr != nil {
+		return formattingErr
+	}
+
+	f, err := os.OpenFile(filepath, os.O_RDWR|os.O_CREATE|os.O_TRUNC, 0755)
```

Comment:
I think the file permissions here should be something like `0644` - a JSON log file probably shouldn't be marked as executable.

Reply:
Makes sense, this was a copy-pasta oversight. Thanks for catching it.